response_operations_ui/views/collection_exercise.py

```diff
@@ -796,6 +816,77 @@ def get_event_name(tag):
     return event_names.get(tag)
 
 
+@collection_exercise_bp.route("/<short_name>/<period>/view-sample-ci", methods=["GET"])
+@login_required
+def get_view_sample_ci(short_name, period):
+    logger.info("Retrieving upload sample collection instrument page", short_name=short_name, period=period)
+    ce_details = build_collection_exercise_details(short_name, period)
+    ce_state = ce_details["collection_exercise"]["state"]
+    ce_details["collection_exercise"]["state"] = map_collection_exercise_state(ce_state)  # NOQA
+    ce_details["eq_ci_selectors"] = filter_eq_ci_selectors(
+        ce_details["eq_ci_selectors"], ce_details["collection_instruments"]
+    )
+
+    locked = ce_state in ("LIVE", "READY_FOR_LIVE", "EXECUTION_STARTED", "VALIDATED", "EXECUTED", "ENDED")
+    _format_ci_file_name(ce_details["collection_instruments"], ce_details["survey"])
+
+    error_json = _get_error_from_session()
+    _delete_sample_data_if_required()
+    show_msg = request.args.get("show_msg")
+
+    success_panel = request.args.get("success_panel")
+    info_panel = request.args.get("info_panel")
+
+    return render_template(
+        "collection_exercise/ce-view-sample-ci.html",
+        ce=ce_details["collection_exercise"],
+        collection_instruments=ce_details["collection_instruments"],
+        error=error_json,
+        locked=locked,
+        sample=ce_details["sample_summary"],
+        survey=ce_details["survey"],
+        success_panel=success_panel,
+        show_msg=show_msg,
+        eq_ci_selectors=ce_details["eq_ci_selectors"],
+        info_panel=info_panel,
+    )
+
+
+@collection_exercise_bp.route("/<short_name>/<period>/upload-sample-file", methods=["GET"])
+@login_required
+def get_upload_sample_file(short_name, period):
+    if not uaa_controller.user_has_permission("surveys.edit"):
+        logger.error("User has insufficient permissions to access this page", user_id=session["user_id"])
+        abort(500)
+    logger.info("Retrieving upload sample file page", short_name=short_name, period=period)
```

Comment:
Not sure these log statements add anything, if we did need that info its in the http request url

Reply:
Removed